test/contracts/apps/recovery_to_vault.js

```diff
@@ -17,22 +20,53 @@ const TokenReturnFalseMock = artifacts.require('TokenReturnFalseMock')
 const TokenReturnMissingMock = artifacts.require('TokenReturnMissingMock')
 const VaultMock = artifacts.require('VaultMock')
 const KernelDepositableMock = artifacts.require('KernelDepositableMock')
+const EthSender = artifacts.require('EthSender')
 
 const APP_ID = hash('stub.aragonpm.test')
 const EMPTY_BYTES = '0x'
-const SEND_ETH_GAS = 31000 // 21k base tx cost + 10k limit on depositable proxies
+const TX_BASE_GAS = 21000
+const SEND_ETH_GAS = TX_BASE_GAS + 10000 // 10k limit on depositable proxies
```

Comment:
I was looking at some of the other tests, and wondering: which branch is this meant to trigger? This is meant to trigger the proxy deposit branch, right? If so, we should probably add a `- 1` here and in the `app/kernel_funds.js` tests to this to make it more clear, since we do a strict less than 10k check in the code.